src/app/(dashboard)/teams/new/data-table.tsx

```diff
@@ -22,38 +19,41 @@ import {
   TableRow,
 } from "@/components/ui/table";
 import { Button } from "@/components/ui/button";
-import { useState } from "react";
 
 interface DataTableProps<TData, TValue> {
   columns: ColumnDef<TData, TValue>[];
   data: TData[];
+  page: number;
+  setPage: (page: number) => void;
+  canGoNext: boolean;
+  canGoPrevious: boolean;
 }
 
 export function DataTable<TData, TValue>({
   columns,
   data,
+  page,
+  setPage,
+  canGoNext,
+  canGoPrevious,
 }: DataTableProps<TData, TValue>) {
-  const [sorting, setSorting] = useState<SortingState>([]);
-  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([]);
-
   const table = useReactTable({
     data,
     columns,
     getCoreRowModel: getCoreRowModel(),
-    getPaginationRowModel: getPaginationRowModel(),
-    onSortingChange: setSorting,
-    getSortedRowModel: getSortedRowModel(),
-    onColumnFiltersChange: setColumnFilters,
-    getFilteredRowModel: getFilteredRowModel(),
-    state: {
-      sorting,
-      columnFilters,
-    },
   });
 
+  function nextPage(page: number) {
+    setPage(page + 1);
+  }
+
+  function previousPage(page: number) {
+    setPage(page - 1);
+  }
+
   return (
     <div>
-      <div className="flex items-center py-2">
+      {/* <div className="flex items-center py-2"> we ll bring this back in new feature
```

Comment:
Let's not leave dead comment on the code, specially now that it is OS and people looking at it would not have context.
We can always just look on GH history to bring back old code

Same for below commented out code